src/fontra/views/editor/editor.js

```diff
@@ -1946,6 +1951,7 @@ export class EditorController {
     for (const overlay of document.querySelectorAll(".cleanable-overlay")) {
       overlay.classList.remove("overlay-layer-hidden");
     }
+    this.setSelectedTool(this.toolIdentifierSelected);
```

Comment:
Then here, we do
```js
    this.setSelectedTool(this. savedSelectedToolIdentifier);
    delete this.savedSelectedToolIdentifier;
```

This in total is a bit more code, but the intention is much much clearer.

Reply:
Done.